app/src/main/java/com/example/solidarity/Event.java

```diff
@@ -15,13 +15,13 @@
 @ParseClassName("Event")
 public class Event extends ParseObject {
 
-    private static final String KEY_DESCRIPTION = "description";
-    private static final String KEY_IMAGE = "image";
-    private static final String KEY_AUTHOR = "author";
-    private static final String KEY_CREATED_KEY = "createdAt";
-    private static final String KEY_TITLE = "title";
-    private static final String KEY_LOCATION = "location";
-    private static final String KEY_DATE = "date";
+    public static final String KEY_DESCRIPTION = "description";
```

Comment:
Why are these made public?

Reply:
I needed to access these variables from the EventsFragment class, but will create getter methods instead and make them private!